<launch>

  <arg name="vehicle_name"   default="H01"/>
  <arg name="map_frame"      default="world"/>
  <arg name="resolution"     default=".15"/>

  <node name="backup_detector" type= "backup_detector_node" pkg="marble_guidance" output="screen">

    <remap from="backup_detector/octomap"       to="merged_map"/>
    <remap from="backup_detector/imu"           to="imu/data"/>
    <remap from="backup_detector/enable_backup" to="enable_backup"/>

    <param name="vehicle_name"                      value="$(arg vehicle_name)"/>
    <param name="map_frame"                         value="$(arg map_frame)"/>
    <param name="loop_rate"                         value="20.0"/>

    <param name="pitch_limit"                       value=".1"/>
    <param name="safety_radius"                     value="0.8"/>
    <param name="z_lower_threshold"                 value=".25"/>
    <param name="resolution"                        value="$(arg resolution)"/>

    <param name="num_query_point_rows"              value="20"/>
    <param name="num_query_point_cols"              value="20"/>
    <param name="num_query_point_layers"            value="15"/>

  </node>

</launch>
